src/index.js

```diff
@@ -1,25 +1,61 @@
 /* eslint-disable import/no-cycle */
 export { default as asInput } from './asInput';
+export { default as Alert } from './Alert';
 export { default as Avatar } from './Avatar';
 export { default as AvatarButton } from './AvatarButton';
+export { default as Badge } from './Badge';
 export { default as Breadcrumb } from './Breadcrumb';
-export { default as Button } from './Button';
+export { default as Button, ButtonGroup, ButtonToolbar } from './Button';
+export {
+  default as Card,
+  CardColumns,
+  CardDeck,
+  CardImg,
+  CardGroup,
+} from './Card';
```

Comment:
Related to the comment above, feels odd that we'd bundle these four other components into the `Card` exports and then unbundle them again here. :) Rather than just, I guess, having separate subdirectories for them all that re-export the react-bootstrap exports.

Reply:
(To be clear, I don't feel strongly about this)

Reply:
I agree it's somewhat weird. The grouping is not arbitrary though. I bundled together subcomponents with their parent. `Card.CardImg` for example. It's possible that we shouldn't export the subcomponents at all. But that would be a breaking change there.